### PR TITLE
Converted page layout update

### PR DESCRIPTION
**Pull request by yoouung**
- add "과정까지 들여다보기" button to show all images
- modify server response to alert error to user

### CODE REVIEW

**Comment by yoouung** [comment]
### screenshot - open `과정까지 들여다보기`
image

### test video

https://github.com/doorooful/CapstoneProject/assets/78146904/13dd6f7c-4b17-42af-a2a1-354478d5c296

**Comment by yoouung** [comment]
### test case for no face detected

https://github.com/doorooful/CapstoneProject/assets/78146904/18a6347b-4d60-4c44-b41c-6c1bc533b4ea

File: dt-demo-be/image.py [hunk 1]
```diff
@@ -68,6 +68,7 @@ def __cropAndSaveFaces(self):
         self.__cropped_faces = modules.cropFaces(args)
     
     def convert(self):
+        # TODO: remove me from line 72 to line 80
```
Comment (doorooful) [inline dt-demo-be/image.py:71]: Please keep following lines, regarding Flask server usage as an API itself.

File: dt-demo-be/app_lab.py [hunk 2]
```diff
@@ -290,7 +290,11 @@ def image_get_conv():
     base64_list = []
 
     for i in img_list:
```
Comment (doorooful) [inline dt-demo-be/app_lab.py:292]: Can you add TODO statement to implement function for fetching information? (i.e. `image_fetch_conv()`)

Comment (doorooful) [reply dt-demo-be/app_lab.py:292]: I think following lines should be implemented there(fetching function).

**Review by doorooful — APPROVED** [review]
Please update upon comments ◡̈

File: dt-demo-be/image.py (hunk 1 above)
Comment (yoouung) [reply dt-demo-be/image.py]: got it! :)

File: dt-demo-be/app_lab.py (hunk 2 above)
Comment (yoouung) [reply dt-demo-be/app_lab.py:293]: ok

**Pull request merged**